app/components/form/index.tsx

```
@@ -4,169 +4,288 @@ import Select from "./select";
 import Submit from "./submit";
 import Text from "./text";
 import TextArea from "./textarea";
+import TextAreaDrop from "./textareaDrop";
 import File from "./file";
-import Array from "./array";
-import type { Parse as ParseType } from "~/parse";
-
-import { Parse } from "~/parse";
+import Array from "./array"
```

Biome (1.9.4)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)